modules/person.rb

```diff
@@ -1,6 +1,6 @@
-require './nameable'
+require_relative './nameable'
 require 'date'
-require './rental'
+require_relative './rental'
 
 class Person < Nameable
```

Comment:
- [ ] Kindly make sure the getters and setters are coming at the first lines in your classes and no in the middle section for readability and cleanliness 
```rb
   attr_accessor :name, :age
  attr_reader :id, :rentals       # Please move these lines on the top in the class
```